.github/workflows/linux_arm64.yml

```diff
@@ -0,0 +1,1517 @@
+          path: ${{ env.BUILD_DIR }}/openvino_developer_package.tar.gz
+          if-no-files-found: 'error'
+
+      - name: Upload openvino debian packages
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: openvino_debian_packages
+          path: ${{ env.BUILD_DIR }}/*.deb
+          if-no-files-found: 'error'
+
+      - name: Upload openvino tests package
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: openvino_tests
+          path: ${{ env.BUILD_DIR }}/openvino_tests.tar.gz
+          if-no-files-found: 'error'
+
+  Debian_Packages:
+    name: Debian Packages
+    needs: Build
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    runs-on: 'aks-linux-16-cores-arm'
+    container:
+      image: 'openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04'
+    env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      DEBIAN_PACKAGES_DIR: /__w/openvino/packages/
+
+    steps:
+      - name: Download OpenVINO debian packages
+        uses: actions/download-artifact@v3
+        with:
+          name: openvino_debian_packages
+          path: ${{ env.DEBIAN_PACKAGES_DIR }}
+
+      - name: Install debian packages & check conflicts
+        run: |
+          apt-get update -y
+          # install our local one
+          apt-get install --no-install-recommends -y dpkg-dev
+          dpkg-scanpackages . /dev/null | gzip -9c > Packages.gz
+          echo "deb [trusted=yes] file:${DEBIAN_PACKAGES_DIR} ./" | tee /etc/apt/sources.list.d/openvino-local.list
+          apt-get update -y
+          apt-get install openvino -y
+        working-directory: ${{ env.DEBIAN_PACKAGES_DIR }}
+
+      - name: Test debian packages
+        run: |
+          /usr/share/openvino/samples/cpp/build_samples.sh
+          /usr/share/openvino/samples/c/build_samples.sh
+          
+          ~/openvino_cpp_samples_build/aarch64/Release/hello_query_device
+          
+          python3 /usr/share/openvino/samples/python/hello_query_device/hello_query_device.py
+          python3 -c 'from openvino import Core; Core().get_property("CPU", "AVAILABLE_DEVICES")'
+          python3 -c 'from openvino import Core; Core().get_property("AUTO", "SUPPORTED_METRICS")'
+          python3 -c 'from openvino import Core; Core().get_property("MULTI", "SUPPORTED_METRICS")'
+          python3 -c 'from openvino import Core; Core().get_property("HETERO", "SUPPORTED_METRICS")'
+          python3 -c 'from openvino import Core; Core().get_property("BATCH", "SUPPORTED_METRICS")'
+          python3 -c 'from openvino.frontend import FrontEndManager; assert len(FrontEndManager().get_available_front_ends()) == 6'
+          benchmark_app --help
+          ovc --help
+
+  Samples:
+    needs: [Build, Smart_CI]
+    timeout-minutes: 20
+    defaults:
+      run:
+        shell: bash
+    runs-on: 'aks-linux-16-cores-arm'
```

Comment:
Andrey B. said that he would be able to create machines with fewer resources on Friday, the 1st.

Reply:
As this workflow would be enabled in post-commit only for now, I think the runner change could be done later on.